Sourcery/Parsing/FileParser.swift

```diff
@@ -234,8 +234,25 @@ extension FileParser {
 
     internal func parseVariable(_ source: [String: SourceKitRepresentable], isStatic: Bool = false) -> Variable? {
         guard let (name, _, accesibility) = parseTypeRequirements(source),
-            accesibility != .private && accesibility != .fileprivate,
-            let type = source[SwiftDocKey.typeName.rawValue] as? String else { return nil }
+            accesibility != .private && accesibility != .fileprivate else { return nil }
+
+        var maybeType: String? = source[SwiftDocKey.typeName.rawValue] as? String
```

Comment:
It will be in the substructure already, right after the entry for the variable declaration. Try `sourcekitten structure --text "var some = String()"` The problem is that it returns it as a separate substructure component, what I didn't notice at first. So maybe it will be not that easy to use, though it looks like we can just check if there is expression call right after variable declaration... Manual parsing is fine, was just thinking if we can avoid doing the job that SourceKit already kind of did for us. Like for typealiases I would prefer to use it's output instead of manually walking through tokens or lines, but for some reason SourceKitten does not provide it =(